semeio/workflows/misfit_preprocessor/job.py

```diff
@@ -6,34 +6,47 @@
 from semeio.workflows.spearman_correlation_job.job import spearman_job
 
 
+def _fetch_scaling_parameters(threshold):
```

Comment:
At this point I think it makes more sense to just update the dict directly with this instead of having this function.

Reply:
Impressive that you catch these important details even after many times seeing the code!